tests/test_ssh_unittest.py

```diff
@@ -67,22 +67,17 @@ def test_master_agent_list_dups(default_config):
         default_config['ssh_key_path'] = tmp.name
         default_config['master_list'] = ['10.10.0.1', '10.10.0.2', '127.0.0.2']
         assert ssh.validate.validate_config(default_config) == {
-            'master_list': 'master_list and agent_list cannot contain duplicates 127.0.0.2'
+            'master_list': 'master_list and agent_list cannot contain duplicates 127.0.0.2',
+            'agent_list': 'master_list and agent_list cannot contain duplicates 127.0.0.2'
```

Comment:
this shouldn't be here (agent_list isn't specified, shouldn't be reporting anything regarding it)